# Topics
map_topic:     "/voxel_map"
target_topic:  "/move_base_simple/goal" # the clicked goal in RViz
mavros_state_topic:   "/mavros/state" 
odom_topic:    "/mavros/local_position/odom"

# Mapping parameters
voxel_width:   0.5
map_bound: [0.0, 100.0, -25, 25, 0.0, 10.0]  # [x_min, x_max, y_min, y_max, z_min, z_max]
dilate_radius: 0.5
# Flight settings
des_height:    1.0

timeout_rrt:  0.02

# Vehicle physical parameters
vehicle_mass:  0.61
grav_acc:      9.81
horiz_drag:    0.70
vert_drag:     0.80
paras_drag:    0.01
speed_eps:     0.0001

max_vel_mag: 4.0
max_bdr_mag: 2.1
max_tilt_angle: 1.05
min_thrust: 2.0
max_thrust: 12.0

chi_vec: [1.0e+4, 1.0e+4, 1.0e+4, 1.0e+4, 1.0e+5]
integral_intervs: 16
weight_t:  20.0
smoothing_eps: 1.0e-2
rel_cost_tol: 1.0e-5






